python/setup.py

```diff
@@ -2,28 +2,15 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # ---------------------------------------------------------
 
-"""Setup file for interpret-community package."""
+"""Setup file for interpret-text package."""
 from setuptools import setup, find_packages
 import os
```

Comment:
Looks like there is a flake8 error here.

Reply:
fixed, done!